tasks/create-cert-webroot.yml

```
@@ -0,0 +1,30 @@
+---
+- name: Check if certificate already exists.
+  stat:
+    path: /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem
```

```suggestion
    path: /etc/letsencrypt/live/{{ _certbot_cert_item.domains | first }}/cert.pem
```

Private variables should be prefixed with `_certbot_`